eng/CredScanSuppression.json

```diff
@@ -2,10 +2,11 @@
     "tool": "Credential Scanner",
     "suppressions": [
         {
-            "file":[
-                "eng/common/testproxy/dotnet-devcert.pfx"
-            ],
-            "_justification": "File contains private key used by test code."
+          "file": [
+            "sdk/keyvault/azure-security-keyvault-certificates/test/ut/recordings/KeyVaultCertificateClientTest.DownloadImportPem.json",
```

Comment:
We should avoid putting entire recording files in the exclusion list. Here is the guidelines that we used for python but apply in general cases as well.

Reply:
Here is the link of guidance in python: https://github.com/Azure/azure-sdk-for-python/blob/main/doc/dev/credscan_process.md

Reply:
Here is the link of guidance in python: https://github.com/Azure/azure-sdk-for-python/blob/main/doc/dev/credscan_process.md